### DISCUSSION TITLE
Api self hosted gate way is not working along with the azure kubernates

### DISCUSSION THREAD

**Discussion by kirankumarsivapuram** [Azure/api-management-self-hosted-gateway#246] [General]
Can any one please help me in solving the below issue
Getting problem while implementing self hosted gate way using Api managment with AKS
Steps followed
1) Created aks cluster
2)created apim in the same resource group
3) created gateway with existed api (echo api)
4)selected the gate way and selected deployment and selected the deployment script Kubernetes
downloaded the given yaml file and ran the download yaml file on aks , after running these steps in api management gateway it should show 1node in gate way .
i) unable to run the pod
ii) unable to see the nodes in the apimanagement service

**Comment by tomkerkhove** [comment]
Duplicate of #247
